xonsh/environ.py

```diff
@@ -2057,6 +2057,15 @@ def __str__(self):
     def __repr__(self):
         return "{0}.{1}(...)".format(self.__class__.__module__, self.__class__.__name__)
 
+    def _repr_pretty_(self, p, cycle):
+        name = "{0}.{1}".format(self.__class__.__module__, self.__class__.__name__)
+        with p.group(0, name + "(", ")"):
```

Comment:
Maybe provide a single space indentation with `p.group(1, ...)`? Makes it clearer IMO

Reply:
I was just restoring the original code, but I agree, that looks nicer.
Also changed the `name` string to an f-string